app/components/windows/settings/OverlaySettings.tsx

```diff
@@ -140,6 +224,61 @@ export default class OverlaySettings extends Vue {
             />
           </div>
         </div>
+        <div class="section">
+          <h1>{$t('Manage Dual Output Scene')}</h1>
+
+          <span>
+            {$t(
+              'The below will create a copy of the active scene collection, set the copy as the active collection, and then apply the function.',
+            )}
+          </span>
+          <div>
+            <h4 style="margin-bottom: 8px;">{$t('Convert to Vanilla Scene')}</h4>
+            <VFormGroup
+              value={this.collection}
+              onInput={(value: string) => {
+                this.collection = value;
+              }}
+              metadata={metadata.list({
+                title: $t('Scene Collection'),
+                name: 'collection',
+                options: this.collectionOptions,
+              })}
+            />
+            <button
+              class="button button--soft-warning"
+              onClick={async () => await this.convertDualOutputCollection()}
+              disabled={this.busy}
+            >
+              {$t('Convert')}
+            </button>
+            {/* <button
```

Comment:
I'm assuming this is disabled, would prefer a small flag on the VM instead of commenting, but again, nitpick territory ;)

Reply:
i believe this is something being worked on but not ready to ship yet